### PR TITLE
Datatask ebbie

### PR DESCRIPTION
**Pull request by ebbie-awino**

### CODE REVIEW

File: DataModels/src/main/java/com/ibm/pmai/models/core/DataPush.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+    public MetadataDetails getMetadataDetails() {
+        return this.metadataDetails;
+    }
+
+    public void setMetadataDetails(MetadataDetails metadataDetails) {
+        this.metadataDetails = metadataDetails;
+    }
+}
+
```
Comment (WanjiruCate) [inline DataModels/src/main/java/com/ibm/pmai/models/core/DataPush.java:90]: Looks great. Having a toString void function is recommended for default object printing. NB: This is just additional, doesn't have to be accomplished this week.

File: DataTask/datapush/uploadJsonToCos/app.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+def allowed_file(filename):
+    return '.' in filename and filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
+# Clean up function to remove files created under the pushed_datasets folder
+def delete_if_exists(url):
+    os.remove(url)  
+
+@api.route('/datapush')
+@api.expect(upload_parser)
+class CollectData(Resource):
```
Comment (WanjiruCate) [inline DataTask/datapush/uploadJsonToCos/app.py:50]: In practice, app.py should not have long definitions. Having a class in another file and creating an instance of the class here is recommended. This is just a recommendation, the PR can be merged as it is now.

**Review by WanjiruCate — APPROVED** [review]

File: DataModels/src/main/java/com/ibm/pmai/models/core/DataPush.java (hunk 1 above)
Comment (ebbie-awino) [reply DataModels/src/main/java/com/ibm/pmai/models/core/DataPush.java:90]: I'll work on it!

File: DataTask/datapush/uploadJsonToCos/app.py (hunk 2 above)
Comment (ebbie-awino) [reply DataTask/datapush/uploadJsonToCos/app.py:50]: I'll work on it!

**Pull request merged**